### DISCUSSION TITLE
wiki: SimultaneousVariantBuilds doesn't work with scons4?

### DISCUSSION THREAD

**Discussion by karlp** [SCons/scons#4028] [Wiki] [answered]
I'm not sure how to address this, but the example given on: https://github.com/SCons/scons/wiki/SimultaneousVariantBuilds doesn't work with scons v4.0.1 (at least)

It simply emits:
```
karlp@strem:~/src/scons-toys$ scons
scons: Reading SConscript files ...

scons: *** Two environments with different actions were specified for the same target: hello.o
(action 1: gcc -o hello.o -c -O2 hello.c)
(action 2: gcc -o hello.o -c -g hello.c)
File "/home/karlp/src/scons-toys/src/SConscript", line 2, in <module>
karlp@strem:~/src/scons-toys$ scons -v
SCons by Steven Knight et al.:
	SCons: v4.0.1, 2020-07-29 00:00:00, by _reproducible on _reproducible
	SCons path: ['/usr/lib/python3.9/site-packages/SCons']
Copyright (c) 2001 - 2020 The SCons Foundation
karlp@strem:~/src/scons-toys$
```

It didn't seem appropriate to make an "issue" for this though, so I've made it a discussion...

**Comment by karlp** [comment, marked as answer]
nevermind, I fixed it after reading something on the cookbook and I can just edit the page directly with a fix :)

**Comment by mwichmann** [comment]
thanks for the update! wikis do have a tendency to "bitrot"... `build_dir` was renamed to `variant_dir` a very long time ago (15+ years) but that doesn't mean all pages got updated. It's only recently that the old deprecated name was actually removed.

**Comment by bdbaddog** [comment]
@karlp - Thanks!
